ledger/heavy/replica/roles.go

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package replica
+
+//go:generate stringer -type=Role
+type Role int
```

Comment:
I still don't understand, why do we need it here? There is only an api on a heavy node. We we have roles there? all the observer functionality should be in the observer repo

Reply:
It's not the only API, but it's also a communication protocol between two or more parties.